### PR TITLE
Remove adminDisabled from report forms

### PR DESCRIPTION
**Pull request by benmartin-coforma**
### Description
**Context:** Admins are not allowed to edit reports, but there are certain forms they _can_ edit, such as the admin banner. All forms on the site are implemented on the same foundation, Form.tsx. Therefore it has to account for both.

**Current state:** We disallow admins from editing reports by marking report forms as `adminDisabled`. Since a report can contain many forms, sometimes in nested routes, we have a function `copyAdminDisabledStatusToForms` which will ensure all forms in a report have the flag set. This is a proper pain in the butt when comparing form templates.

**This PR's change:** We will invert this flag to `editableByAdmins`, and put it on the special admin forms instead. This allows us to delete all of the special handling from the report form templates.

<sup>Note: I changed mcpar.json and mlr.json in app-api, but not the copies in ui-src, because those copies will soon disappear thanks to ~11406~ #11408. _Edit: Done!_</sup>

### Related ticket(s)
n/a

---
### How to test
* Log in as an admin. Ensure you can edit the admin banner, but not reports.
* Log in as a regular user. Ensure you can create a report and enter data.

### Important updates
n/a

---
### Author checklist
- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [x] I have updated relevant documentation, if necessary

### CODE REVIEW

**Comment by benmartin-coforma** [comment]
Currently blocked on #11406 - this PR's unit tests are failing because form templates have not yet been completely removed. Once that PR merges, this one can pull in `main`, and the failing test will disappear ✨

File: services/ui-src/src/components/reports/ReportPageFooter.tsx
Comment (benmartin-coforma) [inline services/ui-src/src/components/reports/ReportPageFooter.tsx:24]: This is wrong! I will fix it & add a unit test.

**Review by benmartin-coforma — COMMENTED** [review]
Whoops. Requesting changes on myself.

**Review by gmrabian — COMMENTED** [review]
yay! love this change. let me know when it's ready for final review!

**Review by karla-vm — APPROVED** [review]

File: services/ui-src/src/components/reports/ReportPageFooter.tsx
Comment (benmartin-coforma) [reply services/ui-src/src/components/reports/ReportPageFooter.tsx]: Donezo.

**Review by gmrabian — APPROVED** [review]
lgtm!

**Pull request merged**
